Analysis/emission_line_fit.py

```diff
@@ -187,17 +176,17 @@ def zoom_gauss_plot(pdf_pages, N, wave, Spect_1D, dispersion, s2, lambda0, worki
             ini_sig1, RMS_pix = rms_func(wave, dispersion, working_wave, y0, o1[1], mask_flag)
             
             # Filling In Arrays
-            flux_g_array[rr] = flux_g 
-            flux_s_array[rr] = flux_s
-            center_array[rr] = o1[0]
-            sigma_array[rr] = o1[1]
-            median_array[rr] = o1[3]
-            norm_array[rr] = max0
-            RMS_array[rr] = ini_sig1
-            SN_array[rr] = (flux_s / ini_sig1)
+            line_dict[col_names[0]][rr] = flux_g                #Flux Gaussian
+            line_dict[col_names[1]][rr] = flux_s                #Flux Observed
+            line_dict[col_names[2]][rr] = (flux_s / ini_sig1)   #S/N
+            line_dict[col_names[3]][rr] = o1[0]                 #Center
+            line_dict[col_names[4]][rr] = max0                  #Norm
+            line_dict[col_names[5]][rr] = o1[3]                 #Median
+            line_dict[col_names[6]][rr] = o1[1]                 #Sigma
+            line_dict[col_names[7]][rr] = ini_sig1              #RMS
```

Comment:
There might be an easier/faster way to use key-value pairs instead of hard-coding each element

Reply:
A simple for loop with `zip` should suffice to simplify